modules/ROOT/pages/traces.adoc

```diff
@@ -0,0 +1,208 @@
+:entities: Agents, brokers, MCP servers, and Mule apps
+:auto-refresh: every 5, 15, or 30 minutes, every hour, or one time per day
+
+Use Traces to track the flow of requests that travel through service pipelines. Get data from the Open Telemetry (OTEL) spans that form your traces. Troubleshoot requests with errors or slow response times, optimize your resource allocation, and track calls end-to-end. Monitor the performance of a specific transaction as it travels through services. 
+
+[[traces-overview]]
+//don't change order of numbers in graphic or calloutlist
+image::traces.png["Span list in Traces Overview"]
+
+[calloutlist]
+. Traces in Anypoint Monitoring
+. <<environment, Environment selector>> for your deployments (such as Sandbox or Production)
+. <<filters, Filter>> field for queries that narrow the list of logs to return
+//{entities} = Agents, Brokers, MCP Servers, and Mule apps
+. List of spans for the supported {entities}
+. Lookback selector for the period of time to capture in the *Span List* (from the default of 15 minutes to the last 7 days)
+. Auto-Refresh {auto-refresh}
+. Manual refresh button
+
+In addition to reviewing traces in Anypoint Monitoring, you can use xref:telemetry-exporter.adoc[Telemetry Exporter] to export traces to third-party observability and analytics platforms.
+
+=== Supported Traces
+
+Traces include spans for:
+
+* Agentic assets: Agent, broker, MCP server instances
+* Flows: Mule flows for the selected task of the specific service
+* Logs: Contextual logs for the selected task
+* Events: Mule runtime events or user actions for the selected service
+
+=== Hosts
+
+Traces in Anypoint Monitoring displays spans from entities deployed to:
+
+* CloudHub 2
+* Runtime Fabric
+
+//TODO: VERIFY with Bhavin 
+
+=== Sampling Rate
+
+Trace data sampling rate:
+
+* 100%: Agents, brokers, and MCP servers
+* 10%: Mule apps and APIs
+
+//TODO: VERIFY ask Bhavin or Sathish
+
+== Before You Begin
+
+Before using traces, ensure that you have:
+
+include::reuse::partial$billing/pricing.adoc[tag=prereqBulletTitanium]
+
+* xref:access-management::permissions-by-product.adoc#monitoring[Permission to view] Mule apps, APIs, and instances of agents, brokers, and MCP servers in Anypoint Monitoring 
+* For apps deployed to CloudHub 2:
+** xref:cloudhub-2::ch2-deploy-shared-space.adoc#enable-trace-data-collection[Enabled trace data collection for your app in a shared space]
+** xref:cloudhub-2::ch2-deploy-private-space.adoc#enable-trace-data-collection[Enabled trace data collection for your app in a private space]
+* For Runtime Fabric, enabled trace data collection on apps deployed to xref:runtime-fabric::use-anypoint-monitoring.adoc#enable-trace-data[Runtime Fabric]
+//* Requirements for Mule apps:
+//TODO: MULE VERSION RELATES TO JUST MULE APPS OR MORE? AND VERIFY VERSION.
+* Mule runtime version 4.6 or later
+//TODO: CONNECTOR RELATES TO JUST MULE APPS OR MORE? AND VERIFY VERSION.
+* HTTP Connector version 1.8 or later
```

Comment:
@bchauhan-mulesoft Do these prerequisites to getting traces in the UI relate to all instances (agentic and non-agentic), maybe just to Mule apps (non-agentic: flows, events, logs), or what?

```suggestion
//* Requirements for Mule apps:
//TODO: MULE VERSION RELATES TO JUST MULE APPS OR MORE? AND VERIFY VERSION.
* Mule runtime version 4.6 or later
//TODO: CONNECTOR RELATES TO JUST MULE APPS OR MORE? AND VERIFY VERSION.
* HTTP Connector version 1.8 or later
```